readme.md

```diff
@@ -0,0 +1,39 @@
+# MariaDB continuous integration infrastructure (MDBCI)
+
+[MDBCI](https://github.com/mariadb-corporation/mdbci) is a set of tools for testing MariaDB components on the wide set of configurations. The main features of MDBCI are:
+
+* automatic creation of virtual machines according to the configuration template,
+* automatic and reliable deploy of MariaDB, Galera, MaxScale and other packages to the created virtual machines,
+* creation and management of virtual machine state snapshots,
+* reliable destruction of created virtual machines.
+
+## Read More information about MDBCI
+
+* [Requirements](docs/requirements.md)
+* [Install MDBCI](docs/quickstart.md)
```

Comment:
The name, the title and the link to the document must be the same.

Here you have link: Install MDBCI, document: quickstart.md, title: Quickstart. They all must be the same.

Why did you split requirements from the installation? Requirements are the part of the installation.